src/pyscf_losc/losc_options.py

```diff
@@ -0,0 +1,157 @@
+                'v2_parameter_c': 1000,
+            }
+        }
+
+    def set_param(self, module, key, value):
+        """Set an option for the LOSC calculation.
+        Parameters
+        ----------
+        module : str
+            The module name of the option.
+        key : str
+            The key name of the option in option module `module`.
+        value :
+            The value name of the option in option module `module`.
+        See also
+        --------
+        Options : all valid options.
+        """
+        module, key = map(str.lower, (module, key))
+        if module not in self._options:
+            raise Exception(
+                f'Wrong module name of LOSC options: module="{module}"')
+        if key not in self._options[module]:
+            raise Exception(
+                f'Wrong key to set options of LOSC: module="{module}" key={key}')
+        self._options[module][key] = value
+
+    def set_params(self, options):
+        """Set one or more options for the LOSC calculation.
+        Parameters
+        ----------
+        options : dict {module: {key: value}}
+            The options dictionary.
+        See also
+        --------
+        Options : all valid options.
+        """
+
+        for module, opts in options.items():
+            module = module.lower()
+            if module not in self._options:
+                raise Exception(
+                    f'Wrong module name of LOSC options: module="{module}"')
+            for key, val in opts.items():
+                key = key.lower()
+                if key not in self._options[module]:
+                    raise Exception(
+                        f'Wrong key to set options of LOSC: module="{module}" key={key}')
+                self._options[module][key] = val
+
+    def get_param(self, module, key):
+        """Get the value of an option.
+        Parameters
+        ----------
+        module : str
+            The module name of the option.
+        key : str
+            The key name of the option in option module `module`.
+        Returns
+        -------
+        value:
+            The value of the option.
+        See also
+        --------
+        Options : all valid options and the return type of the value.
+        """
+        module, key = map(str.lower, (module, key))
+        try:
+            rst = self._options[module][key]
+        except Exception:
+            raise Exception(
+                f'Unknown module or key to get the LOSC option: module="{module}" key={key}')
+        return rst
+
+    def __str__(self):
+        """Print all LOSC options to PySCF output file."""
+        rst = []
+        rst.append('')
+        rst.append('==> LOSC Settings <==')
+        for module, opts in self._options.items():
+            rst.append(f'--> module={module}')
+            for key, val in opts.items():
+                rst.append(f'    {key}: {val}')
+            rst.append('')
+        return '\n'.join(rst)
```

Comment:
Configure your editor to always insert a newline, even if it is the end of file. This happens to all new files you created in this commit.
If using vscode, configure in this way:
```json
"files.insertFinalNewline": true,
```
Some others I use along with this setting:
```json
"files.trimTrailingWhitespace": true,
"files.trimFinalNewlines": true,
```

Or did you use any code formatter? If so, find corresponding configurations for that.